hexrd/instrument/detector.py

```diff
@@ -735,7 +732,6 @@ def config_dict(self, chi=0, tvec=ct.zeros_3,
                 # !!! can't write None to hdf5; substitute with zeros
                 panel_buffer = np.r_[0., 0.]
             '''
-            pass
```

Comment:
Removing this pass would raise errors, please either do not remove it, or handle the function behavior differently.

Reply:
Interestingly, the way the code is commented out, I guess Python would evaluate the multiline string and not throw an error. But this is not good practice - the commented out code should be removed here and either replaced with something meaningful or a pass.